### PR TITLE
Remove the synchronous find-refs window.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Reverts dotnet/roslyn#17789

This can't go in until TS's work happens: https://devdiv.visualstudio.com/DevDiv/_git/TypeScript/pullrequest/56627

and makes its way to vsuml.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide @vladima @minestarks

File: src/VisualStudio/Core/Def/Implementation/Library/FindResults/LibraryManager.cs [hunk 1]
```diff
@@ -1,114 +0,0 @@
...
-using System.Runtime.InteropServices;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Navigation;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.Shell.Interop;
-
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.Library.FindResults
-{
-    [Guid(Guids.RoslynLibraryIdString)]
-    internal partial class LibraryManager : AbstractLibraryManager
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/Library/FindResults/LibraryManager.cs]: Does this get used at all for some object browser stuff?

File: src/VisualStudio/Core/Test/FindResults/FindResultsTests.vb [hunk 2]
```diff
@@ -1,261 +0,0 @@
...
-Imports Microsoft.VisualStudio.LanguageServices.Implementation.Library.FindResults
-Imports Microsoft.VisualStudio.LanguageServices.UnitTests.ObjectBrowser.Mocks
-Imports Roslyn.Test.Utilities
-Imports Roslyn.Utilities
-
-Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.FindResults
-    Public Class FindResultsTests
-        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
-        <WorkItem(1138943, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1138943")>
-        Public Async Function ConstructorReferencesShouldNotAppearUnderClassNodeInCSharp() As System.Threading.Tasks.Task
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Test/FindResults/FindResultsTests.vb]: Do we have similar tests for this for the new one?

**Comment by CyrusNajmabadi** [comment]
Closing in favor of https://github.com/dotnet/roslyn/pull/20905

**Pull request closed without merging**